Activities/GameActivity.cpp

```diff
@@ -372,6 +372,41 @@ void GameActivity::SetCPUTeam(int team)
 */
 }
 
+/// <summary>
```

Comment:
For our .cpp files we use the following to separate methods, please replace these comments with
```/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////```